### ISSUE TITLE
Allow user to resize modal

### ISSUE THREAD

**Issue by ark-it** [StephanWagner/jBox#115]
Seems usefull to have the possibility for the user to resize the modal size. Just droping this as a suggestion, not sure what the implications would be. In addition I really like how smoothly the modal behaves.

**Comment by StephanWagner** [comment]
At least for Desktop it would be a nice addition. I'll see if I get around to implement it. Might be a while though, I have a couple of more important TODOs for jBox at the moment.

**Comment by nikhamidi97** [comment]
Does this enhancement had been applied?

**Comment by StephanWagner** [comment]
No, it's actually quite tricky to implement. I'm pretty busy with other work at the moment, so I can't tell when I get around to do it.

**Comment by redtopia** [comment]
+1
